src/components/typeahead/typeahead.component.ts

```diff
@@ -88,7 +89,24 @@ export class TypeaheadComponent implements AfterViewInit, OnChanges, OnDestroy {
 
         this.loadOptionsCallback = (pageNum: number, pageSize: number, filter: any) => {

```

Comment:
In the docs site, if i go to the Select component and enable paging I get console errors:

```
TypeError: Cannot read property 'key' of null
```

Reply:
Also, in tag inputs, if I `Enable Typeahead` and then click on 'selectFirst' I also get this error.